examples/life.js

```diff
@@ -0,0 +1,119 @@
+qKit.init({border:{width:8,color:'#824222'}});
+
+const suffusion	=192;
+
+var humor=new Array(2);

```

Comment:
I think this can be `const`

Reply:
Yes.  The humor[] array has fixed values.